pkg/variantregistry/snapshot.yaml

```diff
@@ -5290,7 +5286,7 @@ periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-interop-mtc-interop-aws
     Installer: ipi
     JobTier: standard
     LayeredProduct: none
-    Network: sdn
+    Network: ovn
```

Comment:
Doesn't look right either, sdn would have been default in 4.14.

Reply:
OVN became default in 4.12

Reply:
My bad I was thinking of when it was removed.